azurerm/data_source_arm_container_registry_test.go

```diff
@@ -0,0 +1,46 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataSourceAzureRMContainerRegistry_basic(t *testing.T) {
+	ri := acctest.RandInt()
+	config := testAccDataSourceAzureRMContainerRegistry_basic(ri)
+
+	dataSourceName := "data.azurerm_container_registry.test"
```

Comment:
re-reviewing this I noticed this was checking the resource (`azurerm_container_registry.test`) rather than the data source (which has the `data.` prefix) - I've pushed a commit to fix this (I hope you don't mind!)

Reply:
Of course not! Thanks for taking the time to review this and making this fix. Sorry for all this little erros

Reply:
not at all - apologies I missed it the first time around; thanks for this contribution, this'll go out in v1.12 :)